openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -508,6 +508,22 @@ data:
                 - config.openshift.io
                 resources:
                 - networks
+                - ingresses
+                verbs:
+                - list
+                - watch
+              - apiGroups:
+                - ""
+                resources:
+                - nodes
+                verbs:
+                - list
+                - watch
+
```

Comment:
nit: empty line :sweat_smile:

Reply:
we get the node to detect minishift. happy to have any other way to do that.

Reply:
@vdemeester We put a new line to separate out sections of perms specifically for openshift to improve readability I guess..  Take a look at the whole file to see if the new lline should be removed.

@jcrossley3 Do you have any suggestions/

Reply:
> we get the node to detect minishift. happy to have any other way to do that.

ohh sneaky :sweat_smile: I guess that's ok (no better idea coming to my mind :sweat_smile: )

Reply:
i agree with @vdemeester. we have newlines at 473 and 505 to denote non-upstream perms. the newline at 522 is superfluous

Reply:
Since this PR is already merged, I take care of it here.    https://github.com/openshift/knative-serving/pull/78